README.md

```diff
@@ -23,6 +25,9 @@ In this example we'll use three strategies:
   + Google ([https://console.developers.google.com/home]())
     + GOOGLE_CLIENT_ID
     + GOOGLE_CLIENT_SECRET
+  + Slack ([https://api.slack.com/applications]())
```

Comment:
I got the syntax wrong, these links don't work.  Could you update this one and the others?  Sorry!